ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -52,14 +52,53 @@ const deleteSelectedFiles = (component, fileNames) => {
     cy.get(component).then(() => {
         fileNames.forEach((fileName) => {
             cy.get(".cmp-adaptiveform-fileinput__filedelete").should('have.attr', 'role', 'button');
-            cy.get(".cmp-adaptiveform-fileinput__filename").contains(fileName).next().find('.cmp-adaptiveform-fileinput__filedelete').click();
+            cy.get(".cmp-adaptiveform-fileinput__filename").contains(fileName).next().click();
```

Comment:
Why is this __filedelete removed ? This might backward compatibility issue

Reply:
@rismehta  '.cmp-adaptiveform-fileinput__filedelete', this was introduced in v2 version of file-input so  v1 test cases were breaking so changed the logic which was implemented in v1 version.